01_GettingStartedWithDataLab/01_JupyterPythonSQL101.ipynb

```diff
@@ -850,6 +922,38 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.imshow(g(X,Y),origin='lower') # makes the origin start at the lower left corner"
```

Comment:
In this case, the image is a symmetric Gaussian, i.e. there's no visible difference between the plotted images with or without `origin='lower'`. If you want to demonstrate this, maybe a) use a Gaussian rotated by some degrees (argument theta), and also, as is so often in astro, use in the 2nd example the standard `plt.imshow(g(X,Y).T,origin='lower')` (i.e. transpose the image as well).